content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/enforce-overview/index.md

```diff
@@ -0,0 +1,102 @@
+
+The Chainguard Enforce Console provides a high-level overview of all your clusters running Chainguard Enforce. 
+
+![Screenshot of the "Clusters" tab in the Chainguard Enforce Console, showing 0% Policy Compliance, 1 failed Policy, and one active  cluster named "kind-sample-cluster" with 0/4 compliance and last seen 44 seconds ago.](clusters-tab.png)
+
+You can also find the policies you've applied to your clusters.
+
+![Screenshot of the "Policies" tab in the Chainguard Enforce Console, showing two policies associated with a group named "tester." The first, named "sample-policy", has violations while the second, named "log4shell-demo-policy," does not have any violations.](policies-tab.png)
+
+For more information on how you can use the Console as part of your policy creation workflow, follow our guide on [How to create policies in the Chainguard Enforce Console](../chainguard-policies-ui/).
+
+
+## Chainguard Enforce policies
+
+After installing Chainguard Enforce onto a Kubernetes cluster, you can create one or more policies that apply to the images running in your cluster. Policies are written in YAML, and define what images the policy applies to as well as the specifications that they must meet in order for the policy to be satisfied. Each Chainguard Policy needs to be associated with a group, and will be effective for that group as well as all the groups descending from it. Each Cluster needs to be associated with a group and will be enforced based on that group’s policies.
+
+To better understand how policies work in Chainguard Enforce, we encourage you to review the [Sigstore Policy Controller documentation](https://docs.sigstore.dev/policy-controller/overview). In particular, you may want to review the Policy Controller [Admission of images documentation](https://docs.sigstore.dev/policy-controller/overview/#admission-of-images) to understand how to admit images through the cluster image policy. Additionally, Chainguard has assembled a [policy catalog](https://console.enforce.dev/policies/catalog) in the Chainguard Enforce Console which provides a number of sample policies you can use or edit to suit your own needs.
+
+
+## Chainguard Enforce for Git
+
+In addition to ensuring that Kubernetes clusters comply with established policies, Chainguard Enforce also supports Git signature verification for GitHub repositories. Chainguard Enforce for Git is built around [Gitsign](https://docs.sigstore.dev/gitsign/overview/) a component of the [Sigstore project](https://www.sigstore.dev/). Gitsign implements keyless Sigstore to sign Git commits with a valid OpenID Connect identity, like a GitHub or Google account. This allows users to sign their Git commits without the need for GPG keys or a complicated configuration.
```

Comment:
```suggestion
In addition to ensuring that Kubernetes clusters comply with established policies, Chainguard Enforce also supports Git signature verification for GitHub repositories. Chainguard Enforce for Git is built around [Gitsign](https://docs.sigstore.dev/gitsign/overview/) a tool that is part of the [Sigstore project](https://www.sigstore.dev/). Gitsign implements keyless Sigstore to sign Git commits with a valid OpenID Connect identity, like a GitHub or Google account. This allows users to sign their Git commits without the need for GPG keys or a complicated configuration.
```

Reply:
This is a nit, just that it is a tool on its own as part of the Sigstore "suite"

Reply:
that's very helpful, I was having a hard time describing it!